src/SwedbankPay.Sdk/Resources/ConsumerResource.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace SwedbankPay.Sdk.Resources
```

Comment:
Please delete the `Resources` namespace and move all classes within it alongside the feature they relate to.